<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAtkPanel</class>
 <widget class="QWidget" name="EditAtkPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="stats_comboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="effect_comboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="effect_value_spinbox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="target_comboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="reach_comboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="value_spinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-99999999</number>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="nb_turns_spinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
